extensions/mssql/src/contracts.ts

```diff
@@ -1697,6 +1697,10 @@ export namespace GetBackupFolderRequest {
 	export const type = new RequestType<GetBackupFolderRequestParams, string, void, void>('admin/getbackupfolder');
 }
 
+export interface RestoreParams extends azdata.RestoreInfo {
+	ownerUri: string;
```

Comment:
Would suggest just adding this to azdata.proposed.d.ts, we've always been adding them in the dataprotocol calls anyways : https://github.com/microsoft/sqlops-dataprotocolclient/blob/df92b533819e00122ab3c1a45a1653efdd077b17/src/main.ts#L1001